docs/en/how-to-receive-funds-using-payable-functions.md

```diff
@@ -0,0 +1,396 @@
+
+## 3. Call the payable function
+
+1. Move back to the [Loom Examples Landing page](http://localhost:8080) and, this time, select `Payable Demo`:
+
+![Select Payable Demo](/developers/img/select-payable-demo.png)
+
+This shows the `Payable Demo` page:
+
+![Payable Demo Page](/developers/img/payable-demo-page.png)
+
+In the above screenshot, note that the balance of the `PayableDemo` contract is `0`.
+
+2. Select the `Test Payable Function` button, and you will be  prompted to sign a message:
+
+Once you sign the message, you'll notice that your Extdev balance has decreased by `0.001` ETH, and the `PayableDemo` contract balance has increased by the same amount:
+
+![Payable Demo Contract Updated Balance](/developers/img/payable-demo-contract-updated-balance.png)
+
+## 4. Withdraw the Funds from Your Contract
+
+In this section, we'll show how you can withdraw the funds from the `PayableDemo` smart contract.
+
+1. In a terminal window, move into the `loom-examples` directory, and then issue the following command:
+
+```
+npm run payable-demo:withdraw-funds
+```
+
+You'll see something like the following:
+
+```Bash
+> eth-signing@1.0.0 payable-demo:withdraw-funds /Users/andrei/Documents/test/loom-examples
+>  node scripts/withdraw-from-payable-demo-contract
+
+0x240F14c57a3F48B277109046FaC35cA09C6F5E2E has withdrawn 0.001 ETH.
+Owner's balance: 0.001
+```
+
+Things to note:
+
+* In the above output, `0x240F14c57a3F48B277109046FaC35cA09C6F5E2E` is the test account we used to deploy the contract. Your address will be different.
+
+* The private key for this account has been generated in step `1.3`, and you can find it in the `loom/loom_private_key` file.
+
+
+## 5. The Payable Demo - Under the Hood
+
+The purpose of the previous sections was to show how you can run our demo. In the next sections, I'll briefly walk you through the code.
+
+
+### 5.1 The `PayableDemo` Smart Contract
+
+The `PayableDemo` smart contract implements two functions- `buySomething` and `withdrawFunds`
+
+Let's take a look at the `buySomething` function:
+
+
+```JavaScript
+function buySomething() external payable {
+  require(msg.value == 0.001 ether, "You must send at least 0.001 ETH");
+  // Implement your logic here
+  emit BuySomething(msg.sender);
+}
+```
+
+Things to note:
+
+* The `payable` keyword makes it possible for you to call a function and send ETH to the contract at the same time.
+* The amount of ETH sent is available in the `msg.value` variable.
+* We intentionally omitted the logic of the `buySomething` function because it's outside the scope of this tutorial. We've left a comment in the code to indicate where you should place your logic.
+
+The purpose of the `withdrawFunds` function is to let the owner withdraw funds from the contract:
+
+```JavaScript
+function withdrawFunds() public onlyOwner {
+  uint balance = address(this).balance;
+  require(balance > 0, "Balance should be > 0.");
+  address payable ownerPayable = address(uint160(owner()));
```

Comment:
This cast can be avoided by simply using `msg.sender.transfer`, `msg.sender` is supposed to be an `address payable` already

Reply:
@enlight Right! Thanks 👍